src/components/EditField/EditField.js

```diff
@@ -0,0 +1,57 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import './EditField.css';
+
+export class EditField extends PureComponent {
+  state = {
+    editingValue: this.props.value,
```

Comment:
более понятнее что за value
```suggestion
    todoTitle: this.props.value,
```